etc/ctrl.with.edge.yml

```diff
@@ -10,10 +10,10 @@ v: 3
 db:                     ${ZITI_DATA}db/ctrl.db
 
 identity:
-  cert:                 ${ZITI_SOURCE}fabric/etc/ca/intermediate/certs/ctrl-client.cert.pem
-  server_cert:          ${ZITI_SOURCE}fabric/etc/ca/intermediate/certs/ctrl-server.cert.pem
-  key:                  ${ZITI_SOURCE}fabric/etc/ca/intermediate/private/ctrl.key.pem
-  ca:                   ${ZITI_SOURCE}fabric/etc/ca/intermediate/certs/ca-chain.cert.pem
+  cert:                 ${ZITI_SOURCE}ziti-fabric/etc/ca/intermediate/certs/ctrl-client.cert.pem
+  server_cert:          ${ZITI_SOURCE}ziti-fabric/etc/ca/intermediate/certs/ctrl-server.cert.pem
+  key:                  ${ZITI_SOURCE}ziti-fabric/etc/ca/intermediate/private/ctrl.key.pem
+  ca:                   ${ZITI_SOURCE}ziti-fabric/etc/ca/intermediate/certs/ca-chain.cert.pem

```

Comment:
This syntax would be much nicer looking if it were:

`${ZITI_SOURCE}/ziti-fabric/...`

Reply:
I agree. It's the way it is b/c if ZITI_SOURCE isn't specified, it evaluates to blank and we end up with a relative path ziti-fabric/etc if there's no slash, or an absolute path /ziti-fabric/etc if there is.

If having the absolute path as a fallback is OK, I can switch it. 
I considered adding logic to check for empty ZITI_SOURCE but there's nothing special about it. Other people might use different values in their config files, so it seemed a little strange to add something to the code for it.

Thoughts?